Exercise5/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Enter the size of the listNumbers: ");
+            int size = Convert.ToInt32(Console.ReadLine());
+            int[] listNumbers = new int[size];
+            Console.WriteLine("Enter the list number: ");
+
+            for (int i = 0; i < size; i++)
+            {
+                listNumbers[i] = Convert.ToInt32(Console.ReadLine());
+            }
+            sort(listNumbers, size);
+
+            for (int i = 0; i < size; i++)
+            {
+                Console.Write(listNumbers[i] + " ");
+            }
+            Console.ReadKey();
+
+        }
+        static void sort(int[] listNumbers, int size)
```

Comment:
You don't need to add `size` here, just need to use `for (int i = 0; i <listNumbers.Length  - 1; i++)`